app/src/main/res/values/themes.xml

```diff
@@ -10,9 +10,11 @@
         <item name="colorSecondaryVariant">@color/teal_700</item>
         <item name="colorOnSecondary">@color/black</item>
         <!-- Status bar color. -->
-        <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimaryVariant</item>
-        <!-- Font family. -->
-        <item name="android:fontFamily">@font/rubik</item>
+        <item name="android:statusBarColor" tools:targetApi="l">@android:color/transparent</item>
+        <!--&lt;!&ndash; Font family. &ndash;&gt;
+        <item name="android:fontFamily">@font/rubik</item>-->
+        <!-- Navigation bar color. -->
+        <item name="android:navigationBarColor">@color/black</item>
```

Comment:
I think that it is not good to make status bar transparent, because now I have no data on my status bar)
For not simply remove it